status-updates/2023/2023-05-19.md

```diff
@@ -0,0 +1,76 @@
+# Friday, May 19, 2023 - 9:00am EEST
+
+## Participants
+
+RăzvanD, Teo, Liza, Vlad, Mihai, Adrian, Ștefan
+
+## Discussions
+
+### Closing Event
+
+RD: Let's organise it on September 14.
+We'll schedule a full-day event.
+We'll first book a larger room, and then attendees will split into breakout rooms on different subjects: quizzes, assignments, etc.
+
+RD: We are looking at around 200 attendees.
```

Comment:
```suggestion
RD: We are looking at around 100 attendees.
```
Wasn't it 100?

Reply:
@razvand ?

Reply:
Oh, yeah, 200 invitees and 100 attendees. I'll fix that.